src/screens/Messages/Conversation/MessagesList.tsx

```diff
@@ -222,15 +248,3 @@ export function MessagesList() {
     </KeyboardAvoidingView>
   )
 }
-
-function useKeyboardVerticalOffset() {
-  const {top: topInset} = useSafeAreaInsets()
-
-  return Platform.select({
-    ios: topInset,
-    // I thought this might be the navigation bar height, but not sure
-    // 25 is just trial and error
-    android: 25,
```

Comment:
@mozzius This `25` actually was giving me too much space 🤔 seems like zero is better?

Reply:
Hmmm what on earth is that value supposed to be then. `25` is definitely right for my Pixel 3a emulator